packages/cgb-scripts/template/src/init.php

```diff
@@ -17,48 +17,48 @@
  * Enqueue Gutenberg block assets for both frontend + backend.
  *
  * `wp-blocks`: includes block type registration and related functions.
+ * `wp-element`: includes the WordPress Element abstraction for describing the structure of your blocks.
+ * `wp-i18n`: To internationalize the block's text.
  *
  * @since 1.0.0
  */
 function <% blockNamePHPLower %>_cgb_block_assets() {
-	// Styles.
+	// Block frontend styles.
```

Comment:
Not sure if it is just for the frontend. Read the function block.

Reply:
Read this ↓



```using the script, style, editor_script, and editor_style block type registration settings. The editor_-prefixed handles will only be enqueued in the context of the editor, while script and style will be enqueued both in the editor and when viewing a post on the front of your site.
```

Reply:
Yep, you're right. Just submitted the commit to correct the mistake.